### PR TITLE
Build timestamp should be the timestamp of the build and not the last git commit

### PR DESCRIPTION
**Pull request by adamretter**

### CODE REVIEW

File: exist-core/src/main/resources-filtered/org/exist/system.properties [hunk 1]
```diff
@@ -24,6 +24,7 @@ vendor=eXist-db Project
 vendor-url=http://exist-db.org
 product-name=eXist
 product-version=${project.version}
-product-build=${build-tstamp}
+product-build=${maven.build.timestamp}
 git-branch=${build-version}
 git-commit=${build-commit}
+git-commit-timestamp=${build-tstamp}
```
Comment (dizzzz) [inline exist-core/src/main/resources-filtered/org/exist/system.properties:30]: confusing variable name :-/

Comment (dizzzz) [reply exist-core/src/main/resources-filtered/org/exist/system.properties:30]: image
are the others OK?
wondering who is setting the variable?

Comment (adamretter) [reply exist-core/src/main/resources-filtered/org/exist/system.properties:30]: @dizzzz Thanks for noticing those other files - I have amended the PR to include additional changes to those files.

**Comment by adamretter** [comment]
@dizzzz I made the changes you requested, is it possible to merge this please?

**Review by reinhapa — APPROVED** [review]

**Pull request merged**